warehouse/query-core/src/test/java/datawave/query/planner/DatePartitionedQueryPlannerTest.java

```diff
@@ -638,7 +635,6 @@ public void testFieldIndexMinThresholdWithSomeNotMeeting() throws Exception {
         expectEvents("20130104", FieldIndexHoleDataIngest.corleoneUID, FieldIndexHoleDataIngest.caponeUID, FieldIndexHoleDataIngest.sopranoUID);
         expectEvents("20130105", FieldIndexHoleDataIngest.corleoneUID, FieldIndexHoleDataIngest.caponeUID, FieldIndexHoleDataIngest.sopranoUID);
 
-        assertQueryResults();
-        assertSubrangesCorrect();
+        assertSubrangesCorrect(assertQueryResults());
```

Comment:
It took a bit of reading to understand how this test was handling asserting the correct. I would like to see the final query plans asserted, just so it's easy to visually inspect the final results. Something like
```
expectPlan("20130101", "UUID == 'uuid'")
expectPlan("20130101", "(_Hole_ = true) && (UUID == 'uuid')")
```
Doesn't have to be done before a merge, but should be done relatively soon.